bodhi/server/scripts/check_policies.py

```diff
@@ -41,9 +41,12 @@ def check():
         .filter(models.Update.status.in_(
                 [models.UpdateStatus.pending, models.UpdateStatus.testing]))
     for update in updates:
+        decision_context = u'bodhi_update_push_testing'
+        if update.status == models.UpdateStatus.testing:
+            decision_context = u'bodhi_update_push_stable'
```

Comment:
I think this code is right, but it is also confusing initially that we'd mix "testing" and "stable" like this. Can we add a comment explaining why we want to check the stable context when it is in testing? Maybe something like this:

```
# We want to know a decision about pending updates going to testing, and about testing updates going to stable.
```